### PR TITLE
[dev] Review V2K documentation

### PR DESCRIPTION
**Pull request by eduardogmisiuk**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR?** (Link to an issue, e.g. #99999)
https://github.com/dojot/dojot/issues/1584

* **Other information**:

### CODE REVIEW

File: connector/mqtt/vernemq/v2k-bridge/README.md [hunk 1]
```diff
@@ -78,18 +82,18 @@ V2K_SDK_PRODUCER_POOL_INTERVAL_MS      | 100           |
...
-V2K_KAFKA_QUEUE_BUFFERING_MAX_MS                | 0.5                |
-V2K_KAFKA_RETRIES                               | 2                  |
-V2K_KAFKA_RETRY_BACKOFF_MS                      | 100                |
-V2K_KAFKA_SOCKET_KEEPALIVE_ENABLE               | false              |
+Key                                             | Default Value       |
+----------------------------------------------- | ------------------- |
+V2K_KAFKA_ACKS                                  | -1                  |
+V2K_KAFKA_BATCH_NUM_MESSAGES                    | 10000               |
+V2K_KAFKA_CLIENT_ID                             | ${V2K_APP_HOSTNAME} |
+V2K_KAFKA_COMPRESSION_CODEC                     | none                |
```
Comment (rascaraficci) [inline connector/mqtt/vernemq/v2k-bridge/README.md:90]: Aren't we using gzip?

**Review by rascaraficci — COMMENTED** [review]
Good job! I left a single comment.

File: connector/mqtt/vernemq/v2k-bridge/README.md (hunk 1 above)
Comment (eduardogmisiuk) [reply connector/mqtt/vernemq/v2k-bridge/README.md:90]: I changed the value with the env var, hadn't changed the default one. Do you think it's better?

Comment (rascaraficci) [reply connector/mqtt/vernemq/v2k-bridge/README.md:90]: I think we could change the default one.

**Review by rascaraficci — APPROVED** [review]
It's good to me!

**Review by thiagodpf — APPROVED** [review]

**Pull request merged**